README.md

```diff
@@ -239,6 +239,36 @@ Please check out these previous works that helped inspire the creation of starsh
     <img width="100" src="https://raw.githubusercontent.com/starship/starship/master/media/icon.png" alt="Starship rocket icon">
 </p>
 
+## Contributors
```

Comment:
I think it would be best if we put the contents of this heading under our existing "Contributing" heading, on line 221.